content/en/contribute/code/releasing.md

```diff
@@ -10,7 +10,34 @@ aliases: >
 
 ## CHT Core
 
-[Create an issue](https://github.com/medic/cht-core/issues/new/choose) for either a Major/Minor or Patch release issue template and assign it to the release manager to follow the process.
+A release is a set of changes that is planned to be used by at least one implementation that uses the CHT.
```

Comment:
This definition seems to say we won't make a release _unless_ we have a project planning to use the new version. That seems ideal, but is planning an upgrade with a project a prerequisite to release a bugfix?

Here's a possible suggestion, but may need further wordsmithing:
```suggestion
A release is a set of code changes bundled together, ideally with at least one deployment of CHT apps ready to make use of it.
```